src/core/agent/services/connectionService.ts

```diff
@@ -218,10 +217,23 @@ class ConnectionService extends AgentService {
     alias?: string,
     groupId?: string
   ): Promise<string> {
-    const result = await this.props.signifyClient
-      .oobis()
-      .get(signifyName, ConnectionService.DEFAULT_ROLE);
+    const result = await this.props.signifyClient.oobis().get(signifyName);
+    if (!result.oobis[0]) {
+      return "";
```

Comment:
We should always have an OOBI here (we make make sure the end role auth is always done), so lets throw an error